deploy/main.tf

```diff
@@ -1,19 +1,20 @@
 locals {
-  region = "us-east-1"
-  docker_image = "470602773899.dkr.ecr.us-east-1.amazonaws.com/balrog/balrog"
+  region             = "us-east-1"
+  docker_image       = "470602773899.dkr.ecr.us-east-1.amazonaws.com/balrog/balrog"
   docker_image_agent = "470602773899.dkr.ecr.us-east-1.amazonaws.com/balrog/agent"
-  key_name = "sam"
-  vpc_id = "vpc-2c11904a"
-  private_subnets = [ "subnet-2a058116", "subnet-9587f6ce", "subnet-6659712f", "subnet-bfafd492" ]
-  public_subnets = [ "subnet-4004807c", "subnet-51587018", "subnet-8a84f5d1", "subnet-bda8d390" ]
-  vpn_sec_group_id = "sg-03d0a12d9dcc230eb"
-  dns_zone_id = "Z1T9VA34W75OR4"
-  dns_name_admin = "balrogadmin.ghosterydev.com"
-  dns_name_ui = "balrogui.ghosterydev.com"
-  db_name = "balrog"
-  db_host = "balrog.cfheundkobhl.us-east-1.rds.amazonaws.com"
-  db_user = "balrogadmin"
-  db_pass = ""
+  key_name           = "sam"
+  vpc_id             = "vpc-2c11904a"
+  private_subnets    = ["subnet-2a058116", "subnet-9587f6ce", "subnet-6659712f", "subnet-bfafd492"]
+  public_subnets     = ["subnet-4004807c", "subnet-51587018", "subnet-8a84f5d1", "subnet-bda8d390"]
+  vpn_sec_group_id   = "sg-03d0a12d9dcc230eb"
+  ci_sec_group_id    = "sg-0998c98e562cd2f1c"
```

Comment:
new security group for jenkins

Reply:
Maybe leave a comment what the configuration of this security group are expected to be. What permissions they grant.

BTW, security groups can be defined in Terraform as well. Perhaps that it is an option.